pkg/plugins/logger.go

```diff
@@ -0,0 +1,32 @@
+package plugins
+
+import (
+	"context"
+
+	"github.com/http-wasm/http-wasm-host-go/api"
+	"github.com/rs/zerolog"
+)
+
+func initWasmLogger(logger *zerolog.Logger) *WasmLogger {
```

Comment:
looks like it does not make big difference if we use zerolog here. The performance is still good.